AK/Error.h

```diff
@@ -21,10 +21,15 @@ namespace AK {
 
 class Error {
 public:
+    // The Variant can't get any smaller than its largest member (errno), so we might as well just specify `int` here.
+    enum CustomError : int {
```

Comment:
Maybe consider a different name here, like SerenityError, since there are too many ways a thing can be "custom".

Reply:
I'll try to come up with something, `SerenityError` somehow doesn't feel that correct either.